docs/administrator/deploy/infrastructure.md

```diff
@@ -84,6 +85,42 @@ In the Administrator Application, this would look similar to the following:
 
 ![Example external Postgres deployment variables for the query service](../../images/deploy-postgres-query.png)
 
+## Configuring an event bridge 
+An external event bridge can be added to an application at deployment time that routes integration and runtime events to an external service. The services that events can be sent to are:
+
+* [Elasticsearch](#elasticsearch)
+* [Kafka](#kafka)
+
+Runtime events are events that are emitted by the [runtime bundle](../../architecture/application.md#runtime-bundle). 
+
+Integration events are events that are emitted by [connectors](../../modeling/connectors/README.md). For example, if using the [email connector](../../modeling/connectors/ootb/email.md), the event bridge will listen to events in the queue named `emailConnector.SEND`. 
```

Comment:
In other to consume the integration events of each connector, the user has to set the following environment variable: `GATEWAY_INTEGRATION_CONSUMER_DEST`, setting the name of each queue separated by commas. e.g.

```
{
"GATEWAY_INTEGRATION_CONSUMER_DEST" : "emailConnector.SEND, textractConnector.EXTRACT, restConnector.POST"
}
```

Reply:
Ah ok, I didn't interpret that very well so have added it in